rocks.thorium.Thorium.yaml

```diff
@@ -0,0 +1,47 @@
+app-id: rocks.thorium.Thorium
+base: org.electronjs.Electron2.BaseApp
+base-version: '24.08'
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+command: thorium
+separate-locales: false
+rename-desktop-file: thorium-browser.desktop
+rename-icon: thorium
+finish-args:
+  - --share=network
+  - --share=ipc
+  - --device=dri
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --talk-name=org.kde.StatusNotifierWatcher
+  - --env=LD_LIBRARY_PATH=/app/bundle/usr/lib:$LD_LIBRARY_PATH
```

Comment:
variable substitution does not work in finish-args, also you should install the libraries to `/app/lib` then no need to set LD_LIBRARY_PATH.

`/app/foobar/usr/` is a bit bogus path